frontend/src/pages/projects/screens/detail/notebooks/utils.ts

```diff
@@ -21,3 +21,21 @@ export const extractAcceleratorResources = (
     identifier: limitsResource?.[0] || requestsResource?.[0],
   };
 };
+
+export const getImageNameSoftwareVersionJSON = (
+  imageSoftwareVersion: string | undefined,
+): NameVersionPair[] => {
+  if (!imageSoftwareVersion) {
+    return [];
+  }
+  return JSON.parse(imageSoftwareVersion);
+};
+
+export const getNameImageSoftwareVersion = (
+  imageSoftwareVersion: string | undefined,
+): string | undefined => {
+  const imageNameSoftwareVersionJSON = getImageNameSoftwareVersionJSON(imageSoftwareVersion);
+  return `${imageNameSoftwareVersionJSON.at(0)?.name ?? ''} ${
+    imageNameSoftwareVersionJSON.at(0)?.version ?? ''
+  }`;
+};
```

Comment:
there are already functions that exist for this

getImageVersionSoftwareString
and 
getImageVersionDependencies